sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -24,26 +24,34 @@ load(
 
 package(default_visibility = ["//visibility:public"])
 
-MSG_STARTING_ROM_EXT = "Starting ROM_EXT"
-
 MSG_BOOT_FAULT = "BFV:"
 
-MSG_STORE_ACCESS_FAULT = "BFV:07495202(?s:.*)BFV:07495202"
+_BFV_TEMPLATE = "{}{}(?s:.*){}{}".format(MSG_BOOT_FAULT, "{0}", MSG_BOOT_FAULT, "{0}")
+
+MSG_STARTING_ROM_EXT = "Starting ROM_EXT"
+
+MSG_STORE_ACCESS_FAULT = _BFV_TEMPLATE.format("07495202")
 
-MSG_ILLEGAL_INSTRUCTION_FAULT = "BFV:02495202(?s:.*)BFV:02495202"
+MSG_ILLEGAL_INSTRUCTION_FAULT = _BFV_TEMPLATE.format("02495202")
 
 MSG_PASS = "PASS!"
 
-MSG_BAD_KEY = "BFV:02535603(?s:.*)BFV:02535603"
+MSG_SIGVERIFY_BAD_KEY = _BFV_TEMPLATE.format("02535603")
 
-SLOTS = [
-    "a",
-    "b",
-]
+MSG_BOOT_POLICY_BAD_IDENTIFIER = _BFV_TEMPLATE.format("0142500d")
+
+MSG_BOOT_POLICY_BAD_LENGTH = _BFV_TEMPLATE.format("0242500d")
+
+MSG_BOOT_POLICY_ROLLBACK = _BFV_TEMPLATE.format("0342500d")
 
-SLOT_A_OFFSET = "0x0"
+MSG_MANIFEST_BAD_ENTRY_POINT = _BFV_TEMPLATE.format("014d410d")
 
-SLOT_B_OFFSET = "0x80000"
+MSG_MANIFEST_BAD_CODE_REGION = _BFV_TEMPLATE.format("024d410d")
+
+SLOTS = {
```

Comment:
should this go in `rules/opentitan.bzl` ?

Reply:
Probably but I just wasn't sure the BFV constants versus the repeating template. I should probably define the constants in `opentitan.bzl` and use them here to define repeated versions. Will do this in a separate PR if you don't mind.

Reply:
sgtm!